.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+### Description
+
+For a bug: Describe the bug and list the steps you used when the issue occurred.
+include any files necessary to reproduce this by zipping them and attaching them
+to this issue. For features, describe what you'd like, how it might be used and
+why.
+
+#### Steps to reproduce | Feature Idea (Delete one)
```

Comment:
I understand what you mean by it being a separator and not a boolean operator, but as we agree `OR` is clearer than `|` or `||`.

Changed